operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -515,6 +515,7 @@ protected V1PodSpec createSpec(TuningParameters tuningParameters) {
             .affinity(getAffinity())
             .nodeSelector(getServerSpec().getNodeSelectors())
             .nodeName(getServerSpec().getNodeName())
+            .serviceAccountName(getServerSpec().getServiceAccountName())
```

Comment:
We need to also add this to the pod spec in JobStepContext.  We've said that the introspector job runs as much as possible just like the admin server.

Reply:
updated. please help to review